src/utils/partials/alert.hbs

```diff
@@ -0,0 +1,23 @@
+<head>
```

Comment:
Alert should be moved to the portal repo

Reply:
+1. Will send another PR for this because I'm working on a different branch now on Key generation with this.